packages/opencensus-core/src/trace/instrumentation/base-plugin.ts

```diff
@@ -39,21 +38,21 @@ export abstract class BasePlugin implements types.Plugin {
   /** The module name */
   protected moduleName: string;
   /** A tracer object. */
-  protected tracer: modelTypes.Tracer;
+  protected tracer!: modelTypes.Tracer;
```

Comment:
Normally I would discourage `!` for properties like this, but I think it is worth it to get the strict null checks enforced.

I think ideally these would be initialized in the constructor for the plugin, but I'm guessing that would involve a bunch more refactoring.

Reply:
Currently users need to pass `tracer` instance, when enabling the plugins.  It would need more work and breaking change in order to initialize same in the constructor.

Reply:
OK, makes sense!